File: app/src/main/java/cn/leetcode/kk/model/TreeNode.java
package cn.leetcode.kk.model;

public class TreeNode {
   public int value;
   public TreeNode left;
   public TreeNode right;

    public TreeNode() {
    }

    public TreeNode(int value) {
        this.value = value;
    }

    public TreeNode(int value, TreeNode left, TreeNode right) {
        this.value = value;
        this.left = left;
        this.right = right;
    }
}

File: app_kotlin/src/main/java/cn/leetcode/kotlin/MeloZhangZhen/model/TreeNode.kt
package cn.leetcode.kotlin.MeloZhangZhen.model

/**
 * 树形表
 * 二叉树 [1,2,2,3,4,4,3] 是对称的。
 *  [1,2,2,null,3,null,3] 则不是镜像对称的:
 *
        1
       / \
      2   2
     / \ / \
    3  4 4  3
 */
class TreeNode(var `val`: Int?, val left: TreeNode?, val right: TreeNode?) {
}